addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -370,6 +374,128 @@ export class TablePlugin extends Plugin {
         }
         this.dependencies.selection.setSelection(selectionToRestore);
     }
+
+    /**
+     * @param {HTMLTableRowElement} row
+     */
+    normalizeRowDimension(row) {
+        const table = closestElement(row, "table");
+        const rows = [...table.rows];
+        const referenceRowHeight = parseFloat(getComputedStyle(row).height);
+        rows.forEach((row) => {
+            if (
+                row.style.height &&
+                Math.abs(parseFloat(row.style.height) - referenceRowHeight) <= 1
+            ) {
+                row.style.height = "";
+            }
+        });
+    }
+
+    /**
+     * @param {HTMLTableRowElement} row
+     */
+    resetRowSize(row) {
+        row.style.height = "";
+        this.normalizeRowDimension(row);
+    }
+
+    /**
+     * @param {HTMLTableElement} table
+     */
+    normalizeColumnDimension(table) {
```

Comment:
If we rename the other function to `normalizeRowHeight` then this one should probably be renamed `normalizeColumnWidth`.